### ISSUE TITLE
`spring-javaformat:apply` cannot be run from the root

### ISSUE THREAD

**Issue by snicoll** [spring-projects/spring-boot#13276]
I don't know if it's a big deal but I've noticed that `mvn spring-javaformat:apply` (or any other goal of the plugin) doesn't work from the root.

It does as soon as the following is added to `~/.m2/settings.xml`

```xml
<pluginGroups>
	<pluginGroup>io.spring.javaformat</pluginGroup>
</pluginGroups>
```

**Comment by philwebb** [comment]
Tempted to live with it in 2.0.x and deal with it in 2.1 where we have milestones to make sure it's not accidentally included in user consumed POMs.

**Comment by snicoll** [comment]
This has been fixed by #13306
